bin/jqf-zest

```diff
@@ -37,5 +37,7 @@ if [ $# -lt 2 ]; then
   exit 1
 fi
 
+export JVM_OPTS="$JVM_OPTS -Djqf.tracing.MATCH_CALLEE_NAMES=true"
```

Comment:
I don't see why this has to be made the default. It will slow down regular runs of Zest. This option can be set by scripts wishing to run Zest with such an option.